### ISSUE TITLE
Multiple Catches Support

### ISSUE THREAD

**Issue by fidraj** [phpmd/phpmd#462]
Multiple catches were introduced in PHP 7.1. https://wiki.php.net/rfc/multiple-catch

PHPMD evaluates following code as invalid:

```php
<?php
 
try {
   // Some code...
} catch (ExceptionType1 | ExceptionType2 $e) {
   // Code to handle the exception
} catch (\Exception $e) {
   // ...
}
```

With error:

```Unexpected token: |```

**Comment by ravage84** [comment]
it's very likely, that this needs to be fixed in [Pdepend](/pdepend/pdepend).

**Comment by fidraj** [comment]
Thanks. I thought so. I'm opening an issue there too.

**Comment by emirb** [comment]
Addressed in https://github.com/pdepend/pdepend/pull/348

**Comment by tvbeek** [comment]
@fidraj can you verify if this is fixed with the latest PHPMD release?

**Comment by kylekatarnls** [comment]
This is fixed in PDepend 2.5.1
